### ISSUE TITLE
Support strings in TEST_CASE()

### ISSUE THREAD

**Issue by rmja** [ThrowTheSwitch/Unity#282]
It would be neat if the TEST_CASE() macro supported string arguments, such as:

```
TEST_CASE( "Test string" )
void test_string( const char* aString )
{
}
```
[This line](https://github.com/ThrowTheSwitch/Unity/blob/master/auto/generate_test_runner.rb#L95) removes the string, so the generated `main()` in the test runner becomes:

```
int main(void)
{
  UnityBegin("test_parameterized.c");
  RUN_TEST(test_string, 37,   );

  return (UnityEnd());
}
```

instead of the desired

```
int main(void)
{
  UnityBegin("test_parameterized.c");
  RUN_TEST(test_string, 37,  "Test string" );

  return (UnityEnd());
}
```

By removing the line, things seems to work, but I guess there is a reason why the line is there in the first place:)

**Comment by rmja** [comment]
Note that my simple remove-the-line fix does not work nice with the case where a `/` is in the `TEST_CASE` string.

**Comment by SyntaxColoring** [comment]
@mvandervoord I think this request can be closed.  It was implemented by PR #380.

**Comment by mvandervoord** [comment]
Thanks!
